src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -132,14 +141,40 @@ function Content(props: {
   onAccessCodeClick: (accessCodeId: string) => void
 }): JSX.Element {
   const { accessCodes, onAccessCodeClick } = props
+  const [filter, setFilter] = useState<AccessCodeFilter | null>(null)
+
+  const accessCodesWithIssues = useMemo(
+    () =>
+      accessCodes.filter((accessCode) => {
+        return accessCode.errors.length > 0 || accessCode.warnings.length > 0
+      }),
```

Comment:
We should be able to move this inside the `filteredAccessCodes` useMemo below since we don't use it anywhere else now.